app/javascript/react/utils/media.ts

```diff
@@ -5,19 +5,28 @@ const sizes = {
   mobile: "768px",
   smallDesktop: "769px",
   desktop: "1024px",
+  mediocreDesktop: "1091px",
+  mediumDesktop: "1240px",
   largeDesktop: "1440px",
+  hugeDesktop: "1620px",
```

Comment:
💪

Reply:
this is actually rough - I wanted to keep as many header elements as I could across as small desktop sizes as possible. We should have coded all of the buttons from the beginning without functionalities, that would save us those weird layout shifts when changing the resolution